src/python/foglamp/sending_process.py

```diff
@@ -658,15 +648,35 @@ def _last_object_id_update(self, new_last_object_id, stream_id):
         try:
             _logger.debug("Last position, sent |{0}| ".format(str(new_last_object_id)))
 
-            sql_cmd = "UPDATE foglamp.streams SET last_object={0}, ts=now()  WHERE id={1}" \
-                .format(new_last_object_id, stream_id)
+            # TODO : to be changed using PayloadBuilder after FOGL-616
+            #
+            # timestamp = datetime.datetime.now(tz=timezone.utc)
+            # payload = payload_builder.PayloadBuilder() \
+            #     .WHERE(['column', '=', stream_id]) \
+            #     .SET(last_object=new_last_object_id) \
+            #     .SET(ts=str(timestamp)) \
+            #     .payload()
+            #
```

Comment:
1. Payload builder supports update. You are not using it correctly (No multiple .SET, it should be a single .SET with multiple key value), Btw, as discussed today, you need not update ts from code. It should be done by DB
2. Reference of FOGL-616 is wrong here. FOGL-616 is bug where storage is updating all value irrespective of where.

Reply:
1. I'm not using the Payload as for the bug FOGL-616, I retried the code  :
            timestamp = datetime.datetime.now(tz=timezone.utc)
            payload = payload_builder.PayloadBuilder() \
                .SET(last_object=new_last_object_id, ts=str(timestamp)) \
                .WHERE(['id', '=', stream_id]) \
                .payload()

            self._storage.update_tbl("streams", payload)

that seems fine to me (I changed with the single SET), but it updates all the rows,  so I'm using the other way for the moment. 
the reference to the FOGL-616 was for this reason, I will switch to use the commented code instead of current one when FOGL-616 will be solved.

2. about the update of ts, I sent the email, we have the Jira task but we need that for the moment to have the functionality working properly.